src/js/mainContainer.js

```diff
@@ -4,12 +4,13 @@ import DayPickerInput from 'react-day-picker/DayPickerInput';
 import {UserName} from './Components/nameInput';
 import {FontSize} from './Components/fontSize';
 import {Styles} from './Components/styles';
-import {Previews} from './Components/prewieW';
```

Comment:
careful on the naming "stop doing the capital case letters just to be unique. IT sometimes conflicts with or defined stuff, it looks sloppy and does not comply to a standard.